runners/samza/src/test/resources/ExpectedDag.json

```diff
@@ -0,0 +1,67 @@
+{
+  "RootNode": [
```

Comment:
Test case does not cover the use case of when BeamIORegistrar is registered. 

We should register one in the unit test to test

Reply:
Done.